incorrect typings

<!--
If you need help with discord.js installation or usage, please go to the discord.js Discord server instead:
  https://discord.gg/bRCvFy9
This issue tracker is only for bug reports and enhancement suggestions.
You won't receive any basic help here.
-->

**Please describe the problem you are having in as much detail as possible:**
So i think the commit [here](https://github.com/discordjs/discord.js/commit/44ff67dc1192e4ca9aa1cf69390d0453c5302f0c) broke something (see error message below)

**Include a reproducible code sample here, if possible:**
```js
// error occurs while running "tsc -p ."
node_modules/discord.js/typings/index.d.ts:1626:19 - error TS2416: Property 'client' in type 'WebhookClient' is not assignable to the same property in base type 'BaseClient & WebhookFields'.
  Type 'this' is not assignable to type 'Client'.
    Type 'WebhookClient' is missing the following properties from type 'Client': actions, _eval, _validateOptions, channels, and 17 more.

1626    public readonly client: this;
```

**Further details:**
- discord.js version: master
- Node.js version: lts/dubnium
- Operating system: elementaryOS Hera
- Priority this issue should have – please be realistic and elaborate if possible:

<!--
If this applies to you, please check the respective checkbox: [ ] becomes [x].
You don't have to modify the text to suit your particular situation – if you want to
elaborate, please do so in the description.
While it's not a requirement to test your issue on the master branch, it would make fixing
the problem a lot easier for us, so please do so if possible.
-->

- [ ] I have also tested the issue on latest master, commit hash:

--- a/typings/index.d.ts
+++ b/typings/index.d.ts
@@ -303,6 +303,13 @@ declare module 'discord.js' {
 		public toString(): string;
 	}
 
+	export interface ActivityOptions {
+		name?: string;
+		url?: string;
+		type?: ActivityType | number;
+		shardID?: number | number[];
+	}
+
 	export class ClientUser extends User {
 		public mfaEnabled: boolean;
 		public verified: boolean;
@@ -1623,7 +1630,6 @@ declare module 'discord.js' {
 	export class WebhookClient extends WebhookMixin(BaseClient) {
 		constructor(id: string, token: string, options?: ClientOptions);
 		public token: string;
-		public readonly client: this;
 	}
 
 	export class WebSocketManager extends EventEmitter {
@@ -1916,13 +1922,6 @@ declare module 'discord.js' {
 		| 'JOIN_REQUEST'
 		| 'SYNC'
 		| 'PLAY';
-
-	interface ActivityOptions {
-		name?: string;
-		url?: string;
-		type?: ActivityType | number;
-		shardID?: number | number[];
-	}
 
 	type ActivityType = 'PLAYING'
 		| 'STREAMING'

--- a/typings/index.d.ts
+++ b/typings/index.d.ts
@@ -303,13 +303,6 @@ declare module 'discord.js' {
 		public toString(): string;
 	}
 
-	export interface ActivityOptions {
-		name?: string;
-		url?: string;
-		type?: ActivityType | number;
-		shardID?: number | number[];
-	}
-
 	export class ClientUser extends User {
 		public mfaEnabled: boolean;
 		public verified: boolean;
@@ -1922,6 +1915,13 @@ declare module 'discord.js' {
 		| 'JOIN_REQUEST'
 		| 'SYNC'
 		| 'PLAY';
+					  
+	interface ActivityOptions {
+		name?: string;
+		url?: string;
+		type?: ActivityType | number;
+		shardID?: number | number[];
+	}
 
 	type ActivityType = 'PLAYING'
 		| 'STREAMING'

--- a/typings/index.d.ts
+++ b/typings/index.d.ts
@@ -1915,7 +1915,7 @@ declare module 'discord.js' {
 		| 'JOIN_REQUEST'
 		| 'SYNC'
 		| 'PLAY';
-					  
+
 	interface ActivityOptions {
 		name?: string;
 		url?: string;

